src/components/AutoCompleteSuggestions/BaseAutoCompleteSuggestions.js

```diff
@@ -43,13 +44,25 @@ const BaseAutoCompleteSuggestions = (props) => {
         </Pressable>
     );
 
-    const rowHeight = measureHeightOfSuggestionRows(props.suggestions.length, props.isSuggestionPickerLarge);
     const innerHeight = CONST.AUTO_COMPLETE_SUGGESTER.ITEM_HEIGHT * props.suggestions.length;
 
+    const animatedStyles = useAnimatedStyle(() => ({
+        ...styles.autoCompleteSuggestionsContainer,
```

Comment:
Move `styles.autoCompleteSuggestionsContainer` out of `useAnimatedStyle`

Reply:
Fixed.